apps/dashboard/src/components/workflow-editor/steps/configure-step-form.tsx

```diff
@@ -308,6 +317,28 @@ export const ConfigureStepForm = (props: ConfigureStepFormProps) => {
             </>
           )}
 
+          {isStepConditionsEnabled && (
+            <>
+              <SidebarContent>
+                <Link to={'./conditions'} relative="path" state={{ stepType: step.type }}>
+                  <Button
+                    variant="secondary"
+                    mode="outline"
+                    className="flex w-full justify-start gap-1.5 text-xs font-medium"
+                  >
+                    <RiGuideFill className="h-4 w-4 text-neutral-600" />
+                    Step Conditions
+                    <span className="ml-auto flex items-center gap-0.5">
+                      <span>0</span>
```

Comment:
For now, I've hardcoded the value until we have the same logic so we can calculate it